server/build.gradle.kts

```diff
@@ -127,20 +126,13 @@ tasks {
         archiveBaseName.set(rootProject.name)
         archiveVersion.set(tachideskVersion)
         archiveClassifier.set(tachideskRevision)
+        destinationDirectory.set(File("$rootDir/server/build"))
     }
 
     test {
         useJUnit()
     }
 
-    withType<ShadowJar> {
-        destinationDirectory.set(File("$rootDir/server/build"))
-    }
-
-    named("run") {
-        dependsOn(":formatKotlin", ":lintKotlin")
```

Comment:
will run still call formatKotlin?

Reply:
Yep, this will be done whenever a CompileKotlin task is run, which is done for any build task